Gruntfile.js

```diff
@@ -14,7 +14,9 @@ const extract = require('jsxgettext-recursive');
 // use this if you want to recursively match all subfolders:
 // 'test/spec/**/*.js'
 module.exports = function (grunt) {
-  'use strict';
+  'use strict'
```

Comment:
`"semi": 0,` rule is disabled for this repo.
We can turn it on if it makes sense.

Generally the fxa-content-esque repos use semicolons and the fxa-server ones dont. It gets tricky deciding on rules in specific repos.